src/Files.App/Filesystem/FilesystemOperations/ShellFilesystemOperations.cs

```diff
@@ -395,19 +350,10 @@ public async Task<IStorageHistory> DeleteItemsAsync(IList<IStorageItemWithPath>
             EventHandler<Dictionary<string, JsonElement>> handler = (s, e) => OnProgressUpdated(s, e, operationID, progress);
             connection.RequestReceived += handler;
 
-            var deleteResult = new ShellOperationResult();
-            var (status, response) = await connection.SendMessageForResponseAsync(new ValueSet()
-            {
-                { "Arguments", "FileOperation" },
-                { "fileop", "DeleteItem" },
-                { "operationID", operationID },
-                { "filepath", string.Join('|', deleleFilePaths) },
-                { "permanently", permanently },
-                { "HWND", NativeWinApiHelper.CoreWindowHandle.ToInt64() }
-            });
-            var result = (FilesystemResult)(status == AppServiceResponseStatus.Success
-                && response.Get("Success", defaultJson).GetBoolean());
-            var shellOpResult = JsonSerializer.Deserialize<ShellOperationResult>(response.Get("Result", defaultJson).GetString());
+            var (success, deleteResult) = await FileOperationsHelpers.DeleteItemAsync(string.Join('|', deleleFilePaths), permanently, NativeWinApiHelper.CoreWindowHandle.ToInt64(), operationID);
```

Comment:
Given that the interop between Files.App and Files.FullTrust is no longer needed, we shouldn't serialize paths to a string anymore. Instead we should pass the `string[]` for paths directly.